aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -200,13 +203,18 @@ def velocity_aggregated(files_to_agg, site_code, input_dir='', output_dir='./',
             else:
                 WCUR[start:end] = np.full(varlen_list[index + 1], np.nan)
                 WCURqc[start:end] = np.full(varlen_list[index + 1], 9)
-            ##calculate depth
+
+            ##calculate depth and add CELL_INDEX
             if 'HEIGHT_ABOVE_SENSOR' in nc.dims:
                 DEPTH[start:end] = (nc.DEPTH - nc.HEIGHT_ABOVE_SENSOR).values.flatten()
                 DEPTHqc[start:end] = np.repeat(nc.DEPTH_quality_control, n_cells).values
+                CELL_INDEX[start:end] = np.array([np.full(n_measurements, cell) for cell in range(n_cells)],
```

Comment:
This gives you the incorrect order of values. The output variables cycle through all the cells for a given timestamp, then the next timestamp, etc... So I think what you want is
```suggestion
                CELL_INDEX[start:end] = np.tile(np.arange(n_cells, dtype=np.uint32))
```

Reply:
(oops, left out the `n_measurements` arg in my initial suggestion - added now)

Reply:
Yes. That is terribly wrong. I fixed and now it works just fine.